tracker/iou_tracker.py

```diff
@@ -57,6 +61,9 @@
 flags.DEFINE_string('optical_flow', None,
                     'The path to the optical flow predictions. This folder '
                     'should contain one folder per sequence.')
+flags.DEFINE_integer('input_channels', 2, 'The number of channels from the '
```

Comment:
Explain in more detail about what you mean by "input_channels" and what does 2 and 3 mean.

Reply:
Done.